### DISCUSSION TITLE
Renovate ignoring access scope?

### DISCUSSION THREAD

**Discussion by domininik** [renovatebot/renovate#12791] [Request Help]
Last Thursday I authorized Renovate and gave it access to **one** of the repositories within the company I work with.
I was surprised when I learned that Renovate created pull requests in **all** the repositories I have access to.
You can imagine that some people were not happy when they discovered tons of pull requests coming suddenly out of nowhere.
It really looked like a serious security issue.
How could this happen?

**Comment by rarkins** [comment]
It's extremely unlikely that Renovate has a bug like this AND github let it happen too. Renovate simply doesn't have access to any private repos on github.com unless GitHub allows it and tells us about it too.

There's some chance GitHub has a bug and converted your "Select repositories" option to "All repositories" - but I'd rate it as very small considering it's not known to have affected anyone else. The most likely explanation is that you selected "All repositories" by accident.

**Reply by domininik** [reply]
I could not confirm it yet but it would be the simplest explanation indeed.
Thanks for the fast response 👍
